lib/manager.js

```diff
@@ -0,0 +1,106 @@
+"use strict";
+
+let Manifest = require("./manifest");
+let { createFile, generateFingerprint, uriJoin } = require("faucet-pipeline-util");
+let path = require("path");
+
+module.exports = class AssetManager {
+	constructor(referenceDir, { manifestConfig, fingerprint, exitOnError } = {}) {
+		this.referenceDir = referenceDir;
+		this.manifestConfig = manifestConfig;
```

Comment:
🙅

* `manifestConfig.file` should enforce relative paths
* `manifestConfig` properties should be listed explicitly
* ... and passed to the `Manifest` constructor below

Reply:
Done: 54a3e2701bd1a82865fdb62bcdb83cc3ecee43b1 (note that this commit is to be squashed into 936199a2f35409da84ffc8151a151877f1ea0a8c)

However, that exposed a conceptual issue: Since we want to maintain the in-memory manifest regardless of whether it's being persisted, `manifest: false` (or omitting `manifest` config altogether) no longer works because `manifest.baseURI` is required either way. So ... `manifest: { file: false, baseURI: "/whatev" }`?